src/index.js

```diff
@@ -8,25 +8,7 @@ const editJsonFile = require('edit-json-file');
 
 
 module.exports = () => {
-  console.log(chalk.magenta('\nnpm2yarn.js...\n'));
```

Comment:
Oh thanks for removing this, copypasta from our previous npm to yarn script 😆

Reply:
np